store/copr/coprocessor.go

```diff
@@ -978,9 +978,19 @@ func (worker *copIteratorWorker) handleCopResponse(bo *Backoffer, rpcCtx *tikv.R
 	}
 	if otherErr := resp.pbResp.GetOtherError(); otherErr != "" {
 		err := errors.Errorf("other error: %s", otherErr)
+
+		kvRanges := task.ranges.ToPBRanges()
```

Comment:
No need to do this. `KeyRanges` has an `At()` function which can get the first and the last key range.

Reply:
Okay, I have modified it.